runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayUtil.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (C) 2017 Kyle Fricilone
+ * <p>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
```

Comment:
Runelite is BSD licensed. The code you submit should be BSD licensed, or at the very least not GPL. I cannot merge GPL licensed code.